### PR TITLE
Created option to select a random image patch for manifold projection

### PR DESCRIPTION
**Pull request by nthndy**

### CODE REVIEW

File: cellx/manifold/projection.py [hunk 1]
```diff
@@ -28,6 +29,7 @@ class ProjectionMethod(CallableEnum):
     SUM = partial(np.sum, axis=0)
     STD = partial(np.std, axis=0)
     FIRST = partial(lambda x: x[0, ...])
+    RAND = partial(lambda x: (x[np.random.choice(len(x[:,...])), ...]))
```
Comment (quantumjot) [inline cellx/manifold/projection.py:32]:
```suggestion
    RANDOM = partial(lambda x: (x[np.random.choice(x.shape[0]), ...]))
```

File: cellx/manifold/projection.py [hunk 2]
```diff
@@ -20,6 +20,7 @@ class ProjectionMethod(CallableEnum):
         * SUM - the per-pixel sum of the images in the bin
         * STD - the per-pixel standard deviation of images in the bin
         * FIRST - the first image in the bin
+        * RAND - random image in the bin
```
Comment (quantumjot) [inline cellx/manifold/projection.py:23]:
```suggestion
        * RANDOM - random image in the bin
```

**Review by quantumjot — CHANGES_REQUESTED** [review]
Looks great @nthndy ! Can you make these changes?

**Review by nthndy — COMMENTED** [review]
done!

File: cellx/manifold/projection.py (hunk 1 above)
Comment (quantumjot) [reply cellx/manifold/projection.py]: Not quite! Can you make this change too?

File: cellx/manifold/projection.py [hunk 3]
```diff
@@ -28,6 +29,7 @@ class ProjectionMethod(CallableEnum):
     SUM = partial(np.sum, axis=0)
     STD = partial(np.std, axis=0)
     FIRST = partial(lambda x: x[0, ...])
+    RANDOM = partial(lambda x: (x[np.random.choice(len(x[:,...])), ...]))
```
Comment (quantumjot) [inline cellx/manifold/projection.py:32]:
```suggestion
    RANDOM = partial(lambda x: x[np.random.choice(x.shape[0]), ...])
```

**Review by quantumjot — CHANGES_REQUESTED** [review]

**Review by quantumjot — APPROVED** [review]

**Pull request merged**